### PR TITLE
add homework1

### PR DESCRIPTION
**Pull request by olemykha**
I added my first homework. All the requirements are met. 
However, I included keyboard input for the name just for fun.

Linter errors:
WPS421 Found wrong function call: input
WPS421 Found wrong function call: print
WPS420 Found wrong keyword: del

input and print are standard input functions. In general, such calls are acceptable.

'del' operator, which may be considered undesirable in certain contexts. My code uses 'del' to delete the 'user_name' variable, and this is permissible, but I can simply omit this operator since memory management in Python is typically automatic. I think this is not an error because... using 'del' was one of your requirements.

### CODE REVIEW

File: homework/homework041223.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"""HOMEWORK1."""
+user_name = input('Введите Ваше имя: ')
```
Comment (antohaUa) [inline homework/homework041223.py:2]: Try to use English phrases even for input. The will be no limitations for your code understanding  by others ( China, India, USA, Spain developers/qa etc)

**Review by antohaUa — APPROVED** [review]
Good

**Comment by antohaUa** [comment]
Lint file inside has : `flake8 --ignore=WPS,DAR $1`  - this means that WPS and DAR error types should be ignored

File: homework/homework041223.py (hunk 1 above)
Comment (olemykha) [reply homework/homework041223.py:2]: You're right!

**Pull request merged**